.github/workflows/nightly_publish.yml

```diff
@@ -77,6 +77,17 @@ jobs:
         with:
           name: mxnet-model-zoo
           path: mxnet/mxnet-model-zoo/build/reports
+      - name: Configure AWS Credentials
+          uses: aws-actions/configure-aws-credentials@v1
+          with:
+            aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+            aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+            aws-region: us-west-2
+      - name: Copy files to S3 with the AWS CLI
+          if: ${{ github.event.inputs.mode == 'snapshot' }}
+          run: |
+            aws s3 cp serving/build/distributions/serving-0.11.0-SNAPSHOT.tar s3://djl-ai/publish/djl-serving-${{ github.event.inputs.mode }}/serving-0.11.0-${{ github.event.inputs.mode }}.tar \
```

Comment:
shouldn't we remove `-SNAPSHOT` if `inputs.mode` is 'release'?

Reply:
so when it gets copied to s3, it'll take the name of the input mode. I can add a new condition for release or staging as per requirement.